cli/src/lib/libDefs.js

```diff
@@ -64,11 +64,23 @@ async function rebaseCacheRepo(verbose?: VerboseOutput) {
 }
 
 /**
+ * Utility wrapper for ensureCacheRepo with an update expiry of 0 hours.
+ */
+async function updateCacheRepo(verbose?: VerboseOutput): Promise<boolean> {
+  try {
```

Comment:
Probably better to just throw and give detailed errors.
That way if there are issues, people can paste helpful debug info in the bug report